io.github.sigmasd.recordapps.yml

```diff
@@ -0,0 +1,52 @@
+app-id: io.github.sigmasd.recordapps
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all # audio + grpahics
+  - --filesystem=xdg-music # for saving recordings
+  - --share=network # local server
+modules:
+  - name: recordapps
+    buildsystem: simple
+    build-commands:
+      - DENORT_BIN=$PWD/denort ./deno compile -o record -A --include dist/ dist/webview.ts
```

Comment:
can you ship https://github.com/sigmaSd/webview_deno/releases/download/0.9.1/libwebview.x86_64.so inside the flatpak instead of downloading at runtime?

Reply:
is that why it needs network permissions?

Reply:
The apps needs network connections because its spawns 2 servers one for the client( html js css) and the other for the API (that does the actual pulse command spawning)

I can indeed embed that library in the package, but the app already requires network permission , so I  thought it doesn't matter that much
If you think its still important  I can do that

Btw would be great if flathub marked apps that used only local connection as more safe, I imagine a lot of apps with this same architecture that uses web apis as front-end would benefit from this, since they don't really require internet,

Reply:
They are served locally at localhost

Reply:
> I can indeed embed that library in the package, but the app already requires network permission , so I thought it doesn't matter that much

Ha yeah downloading random binaries at runtime is the most secure practice in software....

Reply:
> what happen if the port is already taken? How does it prevent something outside of the sandbox to connect to it?

It uses port 0, so it should use any free port available,
Any outside program can connect to it if it knows the port

Reply:
I updated the package to embed the ffi library